package.json

```diff
@@ -5,13 +5,29 @@
   "main": "index.js",
   "scripts": {
     "build": "sass src/scss:dist/css",
-    "start": "sass --watch src/scss:dist/css & live-server"
+    "start": "sass --watch src/scss:dist/css & live-server",
```

Comment:
I'm not personally a fan of the live-server run, because it keeps constantly reloading. You've added this in the readme file, so i ran it without checking first. I would prefer it either not being a part of the command or not being a part of the readme instructions :)